File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex4.java
//Importando Scanner
import java.util.Scanner;

public class Ex4{
	
	public static void main(String[] args){
		
	//Instanciando Scanner
	Scanner input = new Scanner(System.in);
	
	System.out.println("Maior número entre três\n\n");
	
	//vars
	int n1, n2, n3;
	
	//Entradas
	System.out.print("Insira o 1° número: ");
	n1 = input.nextInt();
	
	System.out.print("Insira o 2° número: ");
	n2 = input.nextInt();
	
	System.out.print("Insira o 3° número: ");
	n3 = input.nextInt();
	
	//Processamento e Saída
	if ( n1>n2 && n1>n3 )
		System.out.println(n1 + " é o maior número");
	else if ( n2>n1 && n2>n3 )
		System.out.println(n2 + " é o maior número");
	else if ( n3>n1 && n3>n2 )
		System.out.println(n3 + " é o maior número");
	
	
		
	}
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex4.c
#include <stdio.h>

main()
{
    printf("Maior numero entre tres \n\n");

    //vars
    int n1, n2, n3;

    //Entrada
    printf("Insira o 1o numero: ");
    scanf("%d",&n1);

    printf("Insira o 2o numero: ");
    scanf("%i",&n2);

    printf("Insira o 3o numero: ");
    scanf("%d",&n3);

    //Processamento
    if ( n1>n2 && n1>n3 )
        printf("%i eh o maior numero", n1);
    else
        if( n2>n1 && n2>n3 )
            printf("%d eh o maior numero", n2);
        else
            if ( n3>n1 && n2>n1 )
                printf("%i eh o maior numero", n3);


}
